packages/react-dom/src/events/plugins/ChangeEventPlugin.js

```diff
@@ -223,15 +223,14 @@ function getTargetInstForInputEventPolyfill(
 /**
  * SECTION: handle `click` event
  */
-function shouldUseClickEvent(elem) {
+function shouldUseClickEvent(elem: Element | Text | null) {
   // Use the `click` event to detect changes to checkbox and radio inputs.
   // This approach works across all browsers, whereas `change` does not fire
   // until `blur` in IE8.
-  const nodeName = elem.nodeName;
+  const nodeName = elem !== null && elem.nodeName.toLowerCase();
   return (
-    nodeName &&
-    nodeName.toLowerCase() === 'input' &&
-    (elem.type === 'checkbox' || elem.type === 'radio')
+    nodeName === 'input' &&
```

Comment:
This doesn't look right. the nodeName would be INPUT here?

Reply:
I moved the lower-casing upwards. Maybe I should rename the variable to `localName` so that it's clear that we're dealing a with the lower-cased version?

Reply:
It's now named `localName` to reflect the casing